README.md

```diff
@@ -1,9 +1,8 @@
 # ActiveStash
 
-ActiveStash is the Rails specific gem for using [CipherStash](https://cipherstash.com).
+[ActiveStash](https://cipherstash.com/activestash) is the Rails-specific gem for using [CipherStash](https://cipherstash.com).
 
-It provides search functionality for ActiveRecord models that are configured to use field level encryption (using [Lockbox](https://github.com/ankane/lockbox) or
-[EncryptedRecord](https://guides.rubyonrails.org/active_record_encryption.html)).
+ActiveStash gives you encryption search on ActiveRecord models using application level encryption (using libraries like [Lockbox](https://github.com/ankane/lockbox) and [ActiveRecord Encryption](https://guides.rubyonrails.org/active_record_encryption.html)).
 
 When records are created or updated, they are indexed into a CipherStash collection
 which can be queried via an ActiveStash::Relation.
```

Comment:
```suggestion
When records are created or updated, they are indexed into a CipherStash collection which can be queried via an `ActiveStash::Relation`.
```

Reply:
Tiny style thing: we make sure we have one newline per sentence. 

Makes it easier for future reviewers to parse mid-sentence changes.